spherical_geometry/tests/test_basic.py

```diff
@@ -184,18 +184,69 @@ def build_polygon(offset):
         assert abs(overlap_area - calculated_area) < 0.0005
 
 
-def test_from_wcs():
+def test_from_wcs_fits_header():
     from astropy.io import fits
 
     filename = os.path.join(ROOT_DIR, 'j8bt06nyq_flt.fits')
     header = fits.getheader(filename, ext=('SCI', 1))
 
     poly = polygon.SphericalPolygon.from_wcs(header)
     for lonlat in poly.to_lonlat():
-        lon = lonlat[0]
-        lat = lonlat[1]
-        assert np.all(np.absolute(lon - 6.027148333333) < 0.2)
-        assert np.all(np.absolute(lat + 72.08351111111) < 0.2)
+        lonlat = np.array(lonlat).T
+        assert_allclose(
+            lonlat,
+            np.repeat([(6.027148333333, -72.08351111111)], repeats=lonlat.shape[0], axis=0),
+            atol=0.2,
+        )
+
+
+@pytest.mark.parametrize(
+    "test_point",
+    [
+        # (0.88955854, 87.53857137), # TODO: fix
+        (20.6543883, 87.60498618),
+        (343.19474696, 85.05565535),
+        (8.94286202, 85.50465173),
+        (27.38417684, 85.03404907),
+        (310.53503934, 88.56749324),
+        # (0, 60), # TODO: fix
+        # (0, 90), # TODO: fix
+        (12, 66),
```

Comment:
some of these test points don't work, but I don't think that's a problem with this PR but rather an issue with polygon construction